src/overlays/actors/ovl_Boss_Mo/z_boss_mo.c

```diff
@@ -22,299 +128,2943 @@ const ActorInit Boss_Mo_InitVars = {
+                        this->csState = MO_DEATH_START;
+                        sMorphaTent1->drawActor = false;
+                        sMorphaTent1->actionState = MO_TENT_DEATH_START;
+                        sMorphaTent1->baseAlpha = 0.0f;
+                        if (sMorphaTent2 != NULL) {
+                            sMorphaTent2->tent2KillTimer = 1;
+                        }
+                        if (player->actor.parent != NULL) {
+                            player->unk_850 = 0x65;
+                            player->actor.parent = NULL;
+                            player->csMode = 0;
+                        }
+                    } else {
+                        this->actor.colChkInfo.health = 1;
+                    }
+                }
+                this->invincibilityTimer = 10;
+            } else if (!(hurtbox->toucher.dmgFlags & 0x00100000) && (hurtbox->toucher.dmgFlags & 0x80)) {
+                if (this->actionState >= MO_CORE_ATTACK) {
+                    func_80078914(&sMorphaTent1->tentTipPos, NA_SE_EN_MOFER_CUT);
+                    sMorphaTent1->cutIndex = this->corePos;
+                    sMorphaTent1->meltIndex = sMorphaTent1->cutIndex + 1;
+                    sMorphaTent1->cutScale = 1.0f;
+                    sMorphaTent1->actionState = MO_TENT_CUT;
+                    sMorphaTent1->timers[0] = 40;
+                    sMorphaTent1->actor.flags &= ~1;
+                    if (player->actor.parent == &sMorphaTent1->actor) {
+                        player->unk_850 = 0x65;
+                        player->actor.parent = NULL;
+                        player->csMode = 0;
+                    }
+                }
+                this->actionState = MO_CORE_STUNNED;
+                this->timers[0] = 30;
+                this->invincibilityTimer = 10;
+                this->actor.speedXZ = 0.0f;
+            }
+            for (i = 0; i < 10; i++) {
+                Vec3f pos;
+                Vec3f velocity;
+
+                velocity.x = Rand_CenteredFloat(4.0f);
+                velocity.y = Rand_ZeroFloat(2.0f) + 3.0f;
+                velocity.z = Rand_CenteredFloat(4.0f);
+                pos = this->actor.world.pos;
+                pos.x += (velocity.x * 3.0f);
+                pos.z += (velocity.z * 3.0f);
+                BossMo_SpawnDroplet(MO_FX_DROPLET, (BossMoEffect*)globalCtx->specialEffects, &pos, &velocity,
+                                    Rand_ZeroFloat(0.08f) + 0.13f);
+            }
+        }
+    }
+    // end !!
+    osSyncPrintf("Core_Damage_check 終わり ！！\n");
+    osSyncPrintf(VT_RST);
+}
+
+void BossMo_Core(BossMo* this, GlobalContext* globalCtx) {
+    static f32 coreBulge[11] = {
+        0.1f, 0.15f, 0.2f, 0.3f, 0.4f, 0.43f, 0.4f, 0.3f, 0.2f, 0.15f, 0.1f,
+    };
+    // Because of the large number of temps, I have indicated which named temps
+    // are not actually on the stack and can be reordered. All pads are unused.
```

Comment:
same here, this level of detail about the stack variables doesnt seem needed anymore if the function is matching